virtualmachine/virtualmachine.go

```diff
@@ -22,7 +22,7 @@ type VirtualMachine interface {
 	Halt() error
 	Start() error
 	GetSSH(ssh.Options) (ssh.Client, error)
-	AddDisk() error
+	AddDisk() ([]string, error)
```

Comment:
Either both AddDisk() & RemoveDisk should take list of disks OR individual disk, for consistency. Unless the code path is very different.